modules/mobile/spec/request/letters_lighthouse_request_spec.rb

```diff
@@ -81,17 +81,32 @@
         end
       end
     end
+  end
+
+  describe 'GET /mobile/v0/letters/beneficiary' do
+    context 'with a valid lighthouse response' do
+      it 'matches the letters beneficiary schema' do
+        VCR.use_cassette('lighthouse_letters/letters_200', match_requests_on: %i[method uri]) do
+          get '/mobile/v0/letters/beneficiary', headers: iam_headers
+          expect(response).to have_http_status(:ok)
+          allow(File).to receive(:read).and_call_original
+          expect(response.body).to match_json_schema('letter_beneficiary')
+        end
+      end
+    end
+  end
 
+  describe 'Error Handling' do
```

Comment:
brought error handling into it's own `describe` since both beneficiary and letters endpoint come from the same upstream endpoint now so their error checking is the same.